.github/workflows/trivy-image-scan.yml

```diff
@@ -105,20 +118,3 @@ jobs:
         with:
           files: |
             ${{ steps.trivy-scan.outputs.scan-results-file }}
-
```

Comment:
Removed as our `telicent-oss/trivy-action` step earlier already does this, I had removed this in PR #103 but think it got reinstated due to a merge conflict with PR #111